recipes-samples/images/rpb-desktop-image.bb

```diff
@@ -1,24 +1,19 @@
-SUMMARY = "Basic X11 graphics image"
-
-IMAGE_FEATURES += "splash package-management debug-tweaks ssh-server-openssh hwcodecs x11 tools-debug"
+require rpb-console-image.bb
 
-LICENSE = "MIT"
-
-inherit core-image distro_features_check extrausers
+SUMMARY = "Basic X11 graphics image"
 
+IMAGE_FEATURES += "x11"
 # let's make sure we have a good image..
-REQUIRED_DISTRO_FEATURES = "x11 pam systemd"
+REQUIRED_DISTRO_FEATURES += "x11"
 
 # make sure we boot to desktop
 # by default and without x11-base in IMAGE_FEATURES we default to multi-user.target
 SYSTEMD_DEFAULT_TARGET = "graphical.target"
 
 CORE_IMAGE_BASE_INSTALL += " \
-    packagegroup-rpb \
     packagegroup-rpb-x11 \
 "
 
-EXTRA_USERS_PARAMS = "\
-useradd -p '' linaro; \
+EXTRA_USERS_PARAMS += "\
```

Comment:
this can be moved into -minimal with the rest of the linaro user config. in fact we can use a camera or h/w video codecs without a display/X11 (headless video transcoding), so it's ok to do it there